apps/coscrad-frontend/src/utils/generic-components/presenters/tables/generic-index-table-presenter/index-table.tsx

```diff
@@ -144,83 +152,107 @@ export const IndexTable = <T extends IBaseViewModel>({
         paginatedData.length === 0 ? (
             <NotFoundPresenter />
         ) : (
-            <div>
-                <table>
-                    <thead>
-                        <tr>
-                            {headingLabels.map(({ headingLabel }) => (
-                                <th key={headingLabel}>{headingLabel}</th>
-                            ))}
-                        </tr>
-                    </thead>
-                    <tbody>
-                        {paginatedData.map((row) => (
-                            <tr key={row.id} data-testid={row.id}>
-                                {headingLabels.map(({ propertyKey }) => (
-                                    // A little inversion of control here
-                                    // We may want to use some currying here
-                                    <td key={String(propertyKey)}>
-                                        {renderCell(row, cellRenderers, propertyKey)}
-                                    </td>
+            <Box sx={{ width: '100%' }}>
+                <Paper>
+                    <TableContainer>
```

Comment:
Please alias this as something other than "TableContainer". That is confusing within the context of our project, where "container" has a reserved meaning.